pkg/codegen/internal/test/testdata/aws-fargate.pp

```diff
@@ -28,7 +28,7 @@
 
 // Create an IAM role that can be used by our service's task.
 resource taskExecRole "aws:iam:Role" {
-	assumeRolePolicy = {
+	assumeRolePolicy = toJSON({
```

Comment:
I haven't found a good way to fix this yet. It should probably be a rewrite that would insert `toJSON` automatically based on type mismatch of the right part vs. left part, but that sounds a bit too complicated for this original version.

Reply:
That's reasonable. Can you file a bug to track enabling this and tag this line with a comment that references the issue?